src/main/kotlin/no/nav/syfo/aktivitetskrav/domain/Aktivitetskrav.kt

```diff
@@ -116,7 +116,11 @@ fun Aktivitetskrav.isAutomatiskOppfylt(): Boolean =
 fun Aktivitetskrav.isNy(): Boolean = this.status == AktivitetskravStatus.NY
 
 fun Aktivitetskrav.isVurdert(): Boolean =
-    this.status != AktivitetskravStatus.AUTOMATISK_OPPFYLT && this.status != AktivitetskravStatus.NY
+    this.status in EnumSet.of(
+        AktivitetskravStatus.UNNTAK,
+        AktivitetskravStatus.OPPFYLT,
+        AktivitetskravStatus.IKKE_OPPFYLT,
+    )
```

Comment:
Her er det noe jeg ikke skjønner! 😕 Hvis status er UNNTAK, vil man ikke da få true både med forrige og ny versjon av denne metoden? 🤔

Reply:
Mja, det er meningen det... Den skal fungere som før bortsett fra at AVVENT skal gi `false`. 
Også endret jeg på uttrykket slik at det sier hvilke statuser som er vurdert i stedet for hvilke som ikke er det.

Reply:
Ja, jeg blandet `AVVENT` og `UNNTAK` 😅 🤷

Reply:
Fort gjort! 😅